core/client/fs/src/test/java/alluxio/client/file/FileInStreamTest.java

```diff
@@ -133,7 +137,7 @@ public void before() throws Exception {
       when(mBlockStore.getEligibleWorkers())
           .thenReturn(Arrays.asList(new BlockWorkerInfo(new WorkerNetAddress(), 0, 0)));
       when(mBlockStore.getInStream(eq((long) i), any(InStreamOptions
-          .class)))
+          .class), any()))
```

Comment:
@bf8086 can you check if the conflict resolution is correct?

Reply:
LGTM.